File: Python/Petshop.py
class Petshop:
    # Atribut
    __ID=""
    __nama=""
    __kategori=""
    __harga=0

    # Constructor
    def __init__(self,ID,nama,kategori,harga):
        self.__ID=ID
        self.__nama=nama
        self.__kategori=kategori
        self.__harga=harga
    
    # Setter
    def set_ID(self,ID):
        self.__ID=ID
    
    def set_nama(self,nama):
        self.__nama=nama
    
    def set_kategori(self,kategori):
        self.__kategori=kategori
    
    def set_harga(self,harga):
        self.__harga=harga
    
    ## Getter
    def get_ID(self):
        return self.__ID
    
    def get_nama(self):
        return self.__nama
    
    def get_kategori(self):
        return self.__kategori
    
    def get_harga(self):
        return self.__harga
    
    # Method
    def tampil(self):
        print("-" * 80)

        # Menampilkan ID
        print(f"{self.__ID}{' ' * (5 - len(self.__ID))} | ", end="")

        # Menampilkan Nama
        print(f"{self.__nama}{' ' * (35 - len(self.__nama))} | ", end="")

        # Menampilkan Kategori
        print(f"{self.__kategori}{' ' * (25 - len(self.__kategori))} | ", end="")

        # Menampilkan Harga
        print(self.__harga)

    

File: PHP/Petshop.php
<?php
class Petshop{
    // Attribute
    private $ID;
    private $nama;
    private $kategori;
    private $harga;
    private $gambar;

    // Constructor
    public function __construct($ID="", $nama="", $kategori="", $harga=0, $gambar=""){
        $this->ID = $ID;
        $this->nama = $nama;
        $this->kategori = $kategori;
        $this->harga = $harga;
        $this->gambar = $gambar;
    }

    // Setter
    public function set_ID($ID){ $this->ID = $ID;}
    public function set_nama($nama){ $this->nama = $nama;}
    public function set_kategori($kategori){$this->kategori = $kategori;}
    public function set_harga($harga){$this->harga = $harga;}
    public function set_gambar($gambar){$this->gambar = $gambar;}
    
    // Getter
    public function get_ID(){ return $this->ID;}   
    public function get_nama(){ return $this->nama;}
    public function get_kategori(){return $this->kategori;}
    public function get_harga(){return $this->harga;}
    public function get_gambar(){return $this->gambar;}

    public function tampil(){
        return "ID: ".$this->ID."<br>Nama: ".$this->nama."<br>Kategori: ".$this->kategori."<br>Harga: ".$this->harga."<br>Gambar: ".$this->gambar;
    }
}
?>